frontend/components/Navbar/Narbar.tsx

```diff
@@ -0,0 +1,31 @@
+import Link from "next/link";
+import styles from "./navbar.module.css";
+
+const Navbar = () => (
+  <div className={styles.background}>
+    <div className={styles.logo}>
+      <h1 className={styles.title}>Portfolio Creator</h1>
+    </div>
+    <div className={styles.middle}>
+      <ul className={styles.navlist}>
+        <li>
+          <Link href="/">Home</Link>
+        </li>
+        <li>
+          <Link href="/portfolio">Templates</Link>
+        </li>
+        <li>
+          <Link href="/about">About</Link>
+        </li>
+      </ul>
+    </div>
+    <div className={styles.right}>
```

Comment:
You can remove the `right` class from this `div` once applying my suggested change about horizontal padding.
```suggestion
    <div>
```